src/rust/engine/rule_graph/src/tests.rs

```diff
@@ -661,7 +694,8 @@ fn full_scale_target() {
       vec![
         DependencyKey::new("InferPythonDependencies"),
         DependencyKey::new("StrippedSourceFiles").provided_params(vec!["SourceFilesRequest"]),
-        DependencyKey::new("PythonModuleOwner").provided_params(vec!["PythonModule"]),
+        DependencyKey::for_known_rule(RuleId::new("map_module_to_address"), "PythonModuleOwner")
```

Comment:
It made more sense to start incorporating this into the existing tests, rather than duplicate these already elaborate tests.

Reply:
This edit is probably unnecessary given your `by_name_simple` test, but no harm done.

Reply:
I thought it would be good to start integrating this into the more elaborate tests. `by_name_simple` is too simple to be the only test.